### PR TITLE
uhd 4.2.0.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

File: Formula/gnupg@2.2.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class GnupgAT22 < Formula
   desc "GNU Pretty Good Privacy (PGP) package"
   homepage "https://gnupg.org/"
-  url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.34.tar.bz2"
-  sha256 "562a3350dcf66cb67c5825c67ff2c2904db1e30ec8e1d353adc14efba9abf43f"
+  url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.35.tar.bz2"
+  sha256 "340bc255938971e6e729b3d9956fa2ef4db8215d77693bf300df2bb302498690"
```
Comment (bayandin) [inline Formula/gnupg@2.2.rb:5]: Do we need to update `gnupg@2.2` in this PR?

Comment (chenrui333) [reply Formula/gnupg@2.2.rb:5]: oh, no, commit cross talk. Let me fix it

**Comment by cho-m** [comment]
```
  uhd:
    * Libraries were compiled with a flat namespace.
      This can cause linker errors due to name collisions, and
      is often due to a bug in detecting the macOS version.
        /opt/homebrew/Cellar/uhd/4.2.0.0/lib/libuhd.4.2.0.dylib
```

**Comment by chenrui333** [comment]
- relates to https://github.com/EttusResearch/uhd/commit/2c7ce2dbf72414b64f8a477be614e23bc12f086d

**Review by Bo98 — DISMISSED** [review]
Thanks for finding that commit.

**Pull request closed without merging**